src/config.ts

```diff
@@ -1,7 +1,7 @@
 // Load config from `.env` file
 require('dotenv').config();
 
-function getEnv(variable: string, defaultValue = undefined): string {
+function getEnv(variable: string, defaultValue = <string | undefined> ''): string {
```

Comment:
You're casting an empty string into a <string | undefined>. What you probably want is:
```suggestion
function getEnv(variable: string, defaultValue: string? = ''): string {
```